Add Academy repos to Frontend Projects table (Fixes #6698)

SDev6699 opened the pull request:
**Description**  
This PR adds the following Academy repositories to the **Frontend Projects** table in the Community Handbook’s Repository Overview page:  

- **academy-theme** — Zihan Kuang, Aabid Sofi  
- **academy-build** — Aabid Sofi  
- **academy-example** — Zihan Kuang  
- **exoscale-academy** — Vacant  
- **layer5-academy** — Vacant  

These entries match the format of existing rows, include maintainer details, use the Layer5 icon as the image, and list the appropriate frameworks (Hugo / Hugo build).  

**Fixes #6697**  

---

**Notes for Reviewers**  
- Frameworks are set to **Hugo** (and **Hugo build / GitHub Action** for academy-build).  
- Maintainers and repo links are based on provided notes from the issue discussion.  
- Image uses the default Layer5 icon (`five`) for consistency.  

**Screenshots**  
_Before:_  
<img width="1470" height="835" alt="before" src="https://github.com/user-attachments/assets/e77fbfaa-09b5-4ea3-a7bf-031b39488250" />


_After:_  
<img width="1470" height="835" alt="after" src="https://github.com/user-attachments/assets/392ca197-0068-4d2d-9c45-4c41419c0869" />

---

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**  
- [x] Yes, I signed my commits.  
**Description**

This PR fixes #

**Notes for Reviewers**


**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.
 

<!--
Thank you for contributing to Layer5 projects! 

Contributing Conventions:

1. Include descriptive PR titles with [<component-name>] prepended.
2. Build and test your changes before submitting a PR. 
3. Sign your commits

By following the community's contribution conventions upfront, the review process will 
be accelerated and your PR merged more quickly.
-->

l5io commented:
🚀 Preview for commit de8dae8f412fd28702a082b0a68cad4a115dee38 at: https://689d0a34c27adf39943ae788--layer5.netlify.app

l5io commented:
🚀 Preview for commit c533077890b52b5bb718c1c0f77ef30f8cf2a108 at: https://689d12201c83a9c95068dcc4--layer5.netlify.app

miacycle reviewed (approved):
Looks great to me! Thanks, @SDev6699 🎉

SDev6699 commented on src/sections/Community/Handbook/repo-data.js:
Hi @miacycle, reverted kanvas-visualize-features.js to match master (accidental formatting earlier). PR now only includes the Academy repos table updates.

l5io commented:
🚀 Preview for commit 34b1bcdafb0f9c2cba4888ce48716e46ed2f1b47 at: https://689d23a199d5bc9bbc6a7cc8--layer5.netlify.app

SDev6699 commented:
Hi @leecalcote @Rajesh-Nagarajan-11 @miacycle,  

I want to clarify why `kanvas-visualize-features.js` appeared in my commit history even though this issue was unrelated to it.  

When I first pushed my changes, I only modified **one file** related to adding the Academy repos in the correct table. You can verify this from my first commit — it shows only one file changed:  
📸 **Screenshot – First Commit (Only 1 file changed)**  
<img width="1470" height="835" alt="Screenshot 2025-08-13 at 9 32 18 PM" src="https://github.com/user-attachments/assets/7069d557-95a9-49ee-98cd-2b43c85b5e5f" />
Commit link: [105fd134](https://github.com/layer5io/layer5/commit/105fd134d2f8eafac8dbde102bfe)  

However, the **CI build failed** due to a linting error in `kanvas-visualize-features.js` that was already present in the repo before my changes.  
📸 **Screenshot – Failing CI Build**  
<img width="1470" height="835" alt="Screenshot 2025-08-13 at 9 36 49 PM" src="https://github.com/user-attachments/assets/60b9b1e8-ed5b-4ebf-91a5-7bffa15d8077" />
CI build link: [Failing Build](https://github.com/layer5io/layer5/actions/runs/16950056313/job/48040528984?pr=6698)  

To make the CI build pass, I ran the lint fix command (`npx eslint src/sections/Kanvas/Kanvas-visualize/kanvas-visualize-features.js --fix`) which automatically adjusted the spacing and changed single quotes to double quotes in that file. You can compare the before and after; the actual logic or functionality in that file remains unchanged, only the formatting was updated.

📸 **Screenshot – Commit after lint fixes**  
<img width="1470" height="835" alt="Screenshot 2025-08-13 at 9 44 50 PM" src="https://github.com/user-attachments/assets/90796d4e-d4eb-45e4-807c-69829c6f05f0" />
Commit link: [4cf7ce95](https://github.com/layer5io/layer5/pull/6698/commits/4cf7ce95b8e3be24d5a96b65ec4ed6b35f02e946)  

📸 **Screenshot – Successful CI build after lint fixes**  
<img width="1470" height="835" alt="Screenshot 2025-08-13 at 9 47 11 PM" src="https://github.com/user-attachments/assets/a4602070-8fcc-4eae-87bb-8a85c22e4f23" />

This formatting change was not part of the original scope but was necessary to get the CI build to pass so the PR could be merged successfully. The main change for this PR is still only the table update for the Academy repos.

Rajesh-Nagarajan-11 commented:
@SDev6699 Kanvas-visualiser-feature.js match to master , so that only 1 file alone changed

SDev6699 commented:
Hi @Rajesh-Nagarajan-11 @leecalcote @miacycle, I have reverted back to the first commit and squashed all the extra commits. The PR now only contains the intended change in repo-data.js.

vr-varad commented:
Thank you for your contribution!
Let's discuss this during the website call today at 5:30 PM IST | 7 AM CT

Add it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit?usp=drivesdk), if you would :)

yash37158 commented:
✅ LGTM, thanks for the cleanup @SDev6699

l5io commented:
🚀 Preview for commit 90c3ae9deb03af884a62a7de25aeda6ecc67e6c3 at: https://68a475ad6b9eff08b4b3a34e--layer5.netlify.app

leecalcote commented:
🎉 **Thank you for your first contribution to Layer5!** 🎉

@SDev6699, we're thrilled to welcome you to the Layer5 community! Your contribution to add the Academy repositories to the Frontend Projects table (#6698) is fantastic and helps keep our community handbook comprehensive and up-to-date.

Your attention to detail in properly formatting the repository entries and including the correct maintainer information shows great care for the community. This kind of contribution helps new contributors understand our project structure better.

We're excited to have you as part of the Layer5 family and look forward to your future contributions! 🚀

Welcome aboard! 🌟
